CacheWarmer/AspectCacheWarmer.php

```diff
@@ -94,7 +93,7 @@ public function warmUp($cacheDir)
                     "/resource=" . $realPath
                 );
             } catch (\Exception $e) {
-                $errors[$realPath] = $e;
+                /* noop */
```

Comment:
I didn't, this is cache warmer, there is no Input/Output here. Check previous code, you had this $errors var, it is filled in, but unused afterwards. It is/was dead code.